[DNS Firewall] Update instructions regarding PTR records on DNS Firewall cluster (#18563)
cloudflare/cloudflare-docs: src/content/docs/dns/dns-firewall/faq.mdx

<Details header="How can I set PTR records for nameserver hostnames?">

> To use PTR records on the assigned DNS Firewall cluster IPs that point to your nameserver hostnames, refer to the following API endpoints:

Reviewer: ```suggestion
To set up PTR records for the DNS Firewall cluster IPs assigned to your nameserver hostnames, use the following API endpoints:
```
